### ISSUE TITLE
Add a max FPS limit control to the options menu

### ISSUE THREAD

**Issue by Abureaucrat** [space-wizards/space-station-14#43626]
### What version did the issue occur in?

wizards/b9a97b9

### Description

Apparently some GPUs don't understand the concept of moderation without the game you're playing telling them there's a frame-per-second limit. Mine are apparently in this category.

With Vsync turned off and in full-screen mode my GPUs push themselves into their non-overclocked limit and in doing so spiking their energy draw, heat to about 70 Celsius, and make a massive amount of hellish noise.

I strongly request the addition of an alternative frame limit should Vsync be disabled as this **can and will** go so far as to physically damage the computers of the unaware and unprepared. 
(Some people play on toasters and they shouldn't suffer this for doing so)

### Reproduction

1) Have a GPU that doesn't have a specific built-in limit just for this. (Thanks Nividia)
2) Play on any server in fullscreen with Vsync off
3) Cover your ears from the infernal screams

### Additional Context

I only realized this was a universal thing to all servers recently when I turned Vsync off and went to test if it was just the one.
On the plus side; my computer took it like a champ apparently. It seems like there's no damage aside from possibly my hearing.

Apparently there was a post on Discord about this exact issue back in 2024 that just never made its way to Git.
There **may** have been similar reports of this. If there are then I couldn't find them; I am not very familiar with Github in general.
Either way, since then everyone that has the issue has collectively decided to just use Vsync.

**Comment by mirrorcult** [comment]
fps limit already exists, see `display.max_fps` cvar

**Comment by Abureaucrat** [comment]
> fps limit already exists, see `display.max_fps` cvar

This is arcane to me. Where do I find this in the settings?

**Comment by mirrorcult** [comment]
you can set it ingame by opening the console (\`) and using the cvar command, i.e. `cvar display.max_fps (whatever fps limit)`

**Comment by Abureaucrat** [comment]
Okay! This certainly works and is effective.

Would it be possible to have this set to a baseline of some sort to prevent those unaware of it from hitting 700 fps?
Also maybe giving it a UI element in the game options menu as a longer term goal?
